paddle/fluid/inference/tests/api/analyzer_resnet50_tester.cc

```diff
@@ -48,7 +48,6 @@ void profile(bool use_mkldnn = false) {
 
   if (use_mkldnn) {
     cfg.EnableMKLDNN();
-    cfg.pass_builder()->AppendPass("fc_mkldnn_pass");
```

Comment:
Please, disable the `fc_mkldnn_pass` for ResNet50 model only.

Reply:
Is there any way to check it?

Reply:
You could add an argument to the app or read an env variable?

Reply:
I have agreed with @wojtuss to control fc_mkldnn_pass through command line parameter in resnet50 analyzer app.